dotCMS/src/main/java/com/dotcms/mock/request/MockHttpRequest.java

```diff
@@ -18,15 +26,36 @@
 public class MockHttpRequest implements MockRequest {
 
     private final HttpServletRequest request;
-
+    private final Map<String,String[]> paramMap;
     public MockHttpRequest(final String hostname, final String uri) {
         HttpServletRequest mockReq = new BaseRequest().request();
         Mockito.when(mockReq.getRequestURI()).thenReturn(uri);
         Mockito.when(mockReq.getRequestURL()).thenReturn(new StringBuffer("http://" + hostname + uri));
         Mockito.when(mockReq.getServerName()).thenReturn(hostname);
         Mockito.when(mockReq.getRemoteAddr()).thenReturn("127.0.0.1");
         Mockito.when(mockReq.getRemoteHost()).thenReturn("127.0.0.1");
+        paramMap = new HashMap<>();
+        if(uri.contains("?")) {
+            final String queryString = uri.substring(uri.indexOf("?") + 1, uri.length());
+            Mockito.when(mockReq.getQueryString()).thenReturn(queryString);
+            List<NameValuePair> additional = URLEncodedUtils.parse(queryString, Charset.forName("UTF-8"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'additional' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5391667)